less/spacing.less

```diff
@@ -1,27 +1,18 @@
-
-.responsive-spacing(panel, @spacing-1-mobile, @spacing-1-tablet, @spacing-1-desktop);
-.responsive-spacing-vertical(section-1, @spacing-1-mobile, @spacing-1-tablet, @spacing-1-desktop);
-.responsive-spacing-vertical(section-2, @spacing-2-mobile, @spacing-2-tablet, @spacing-2-desktop);
-.responsive-spacing-vertical(section-3, @spacing-3-mobile, @spacing-3-tablet, @spacing-3-desktop);
-
 .spacing(0, 0);
 .spacing(1, @spacer);
 .spacing(2, (2 * @spacer));
 .spacing(3, (3 * @spacer));
+.spacing(4, (4 * @spacer));
+.spacing(5, (5 * @spacer));
```

Comment:
Is this change necessary?  I appreciate it looks cleaner, but it generates a lot more classes, I can't see any reason for us to need larger horizontal spacing, at that point we should be using grids for alignment.

Reply:
I find very useful this horizontal spacing in the new designs when I need more air between columns or cells.
![image](https://user-images.githubusercontent.com/1814752/32269958-f0d01a72-beeb-11e7-892a-e48be24679e8.png)
![image](https://user-images.githubusercontent.com/1814752/32269995-16a248e2-beec-11e7-868f-2aadbf38cb48.png)

Reply:
I don't know, this doesn't feel quite right to me.  We have a grid for for vertical alignment, this is effectively ignoring it.

Reply:
Removed :)